Src/Support/Google.Apis.Auth/OAuth2/DefaultCredentialProvider.cs

```diff
@@ -82,70 +82,77 @@ public DefaultCredentialProvider()
         /// <summary>Creates a new default credential.</summary>
         private async Task<GoogleCredential> CreateDefaultCredentialAsync()
         {
-            // 1. First try the environment variable.
-            string credentialPath = GetEnvironmentVariable(CredentialEnvironmentVariable);
-            if (!string.IsNullOrWhiteSpace(credentialPath))
+            GoogleCredential credential =
+                // 1. First try the environment variable.
+                await GetADCFromEnvironmentVariableAsync().ConfigureAwait(false)
```

Comment:
Very minor nit that can definitely be ignored for now - I'd use GetAdcFromEnvironmentVariable (so ADC => Adc).
It's only local methods though, so trivial to change later.

Reply:
Done on all